Extensions/Versioning/VersionDotNetCoreAssembliesTask/task/task.json

```diff
@@ -101,7 +101,7 @@
     }
       
    ],
-  "minimumAgentVersion": "1.82.0",
+  "minimumAgentVersion": "1.95.1",
```

Comment:
According to the [docs for minimumAgentVersion](https://github.com/Microsoft/azure-pipelines-task-lib/blob/master/node/docs/minagent.md), node is only present in this version (TFS 2015 Update 2) or later.